File: migration/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int input;

int get_positive_int(void);

int main(void)

{
    input = get_positive_int();
    // i to make new line
    for (int i = 0; i < input; i++)
    {
        // k to print space
        for (int k = 1; k < input - i; k++)
        {
            printf(" ");
        }
        //j to print #
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

// this function to get input from 1 to 8
int get_positive_int(void)
{
    int i;
    do
    {
        i = get_int("Height: ");
    }
    while (i < 1 || i > 8);
    return i;
}

File: sentimental-mario-less/mario.py
def main():
    height = get_height()
    for i in range(height):
        # to print space
        for k in range(height - i - 1):
            print(' ', end="")
        # to print # and do not break to new line
        for l in range(i + 1):
            print("#", end="")
        # to print a new line
        print("")


def get_height():
    while True:
        try:
            n = int(input("Height: "))
            if n > 0 and n < 9:
                break
        except ValueError:
            print("That's not an integer!")
    return n


main()

